<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1319</width>
    <height>766</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="mainVLayout">
    <item>
     <widget class="QTabWidget" name="mainTabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="inventoryTab">
       <attribute name="title">
        <string>Player Inventory</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="playerInfoVLayout">
          <item>
           <layout class="QHBoxLayout" name="playerNameHLayout">
            <item>
             <widget class="QLabel" name="playerNameLabel">
              <property name="text">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Player Name: &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="weightHLayout">
            <item>
             <widget class="QLabel" name="weightLabel">
              <property name="text">
               <string>Weight Carried: &lt;int&gt;</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="coinsHLayout">
            <item>
             <widget class="QLabel" name="coinsLabel">
              <property name="text">
               <string>Coins: &lt;int&gt;</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="healthHLayout">
            <item>
             <widget class="QLabel" name="healthLabel">
              <property name="text">
               <string>Health:</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Fixed</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QProgressBar" name="healthProgBar">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>350</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>750</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="value">
               <number>24</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="staminaHLayout">
            <item>
             <widget class="QLabel" name="staminaLabel">
              <property name="text">
               <string>Stamina:</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Fixed</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>29</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QProgressBar" name="staminaProgBar">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>350</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>750</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="value">
               <number>24</number>
              </property>
              <property name="textVisible">
               <bool>true</bool>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="invertedAppearance">
               <bool>false</bool>
              </property>
              <property name="format">
               <string>%p%</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="inventoryVLayout">
          <item>
           <widget class="QLabel" name="inventoryLabel">
            <property name="text">
             <string>Player Inventory</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="itemKindHLayout">
            <item>
             <widget class="QLabel" name="itemKindLabel">
              <property name="text">
               <string>Item Type:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="itemKindComboBox_pInv">
              <property name="currentText">
               <string>&lt;All Items&gt;</string>
              </property>
              <item>
               <property name="text">
                <string>&lt;All Items&gt;</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Weapon</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Armor</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Usable</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Misc.</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QTableView" name="mainInvTableView">
            <property name="autoFillBackground">
             <bool>true</bool>
            </property>
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="buySellTab">
       <attribute name="title">
        <string>Buy/Sell with NPC</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="pNameLabel_buySell">
            <property name="text">
             <string>&lt;user pName&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Coins: &lt;int&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Weight Carried: &lt;int&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTableView" name="pInvTableView_buySell"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="npcNameLabel_buySell">
            <property name="text">
             <string>&lt;NPC pName&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Coins: &lt;int&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTableView" name="npcInvTableView_buySell"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
